impala/hiveserver2.py

```diff
@@ -481,7 +481,9 @@ def fetchone(self):
 
     def fetchcbatch(self):
         '''Return a CBatch object of any data currently in the buffer or
```

Comment:
nit: not your change, but could prettify the grammar of the existing comment.

Reply:
Done